src/azure-cli/requirements.py3.windows.txt

```diff
@@ -75,7 +75,7 @@ azure-mgmt-sql==0.13.0
 azure-mgmt-sqlvirtualmachine==0.4.0
 azure-mgmt-storage==5.0.0
 azure-mgmt-trafficmanager==0.51.0
-azure-mgmt-web==0.42.0
+azure-mgmt-web==0.43.1
```

Comment:
shouldn't this match the sdk version? 0.44.0?

Reply:
done